friends/src/FriendForm.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import './FriendForm.css';
+import axios from 'axios';
+
+class FriendForm extends React.Component {
+  state = {
+    friend: {
+      name: '',
+      age: '',
+      email: ''
+    }
+  }
+
+  changeHandler = e => {
+    e.persist();
+    this.setState(prevState => ({
+      friend: {
+        ...prevState.friend,
+        [e.target.name]: e.target.value
+      }
+    }));
+    console.log(this.state);
+  }
+
+  addFriend = (e, item) => {
+    axios.post('http://localhost:5000/friends', item)
+    .then(res => {
+      console.log(res)
+    })
+    .catch(err => {
+      console.log(err)
+    })
+  }
```

Comment:
Right now, this only works because the default form behavior of refreshing a page upon submit is not being stopped. Otherwise the updated data here has no way of telling the the friendlist in the parent component that there is updated data from the API. This function should be moved back up to App, prevent the default of the form refreshing, and you can invoke a function that gets the list of friends again once completed to tell the page it will need to render new data.